aspnetcore/security/data-protection/configuration/overview.md

```diff
@@ -129,7 +129,7 @@ public void ConfigureServices(IServiceCollection services)
 
 ## SetApplicationName
 
-By default, the Data Protection system isolates apps from one another, even if they're sharing the same physical key repository. This prevents the apps from understanding each other's protected payloads.
+By default, the Data Protection system isolates apps from one another based on their content root path, even if they're sharing the same physical key repository. This prevents the apps from understanding each other's protected payloads.
```

Comment:
Since the apps don't share the same content root path, make "path" plural ...

```suggestion
By default, the Data Protection system isolates apps from one another based on their content root paths, even if they're sharing the same physical key repository. This prevents the apps from understanding each other's protected payloads.
```